modules/node_modules/@ncigdc/components/File.js

```diff
@@ -13,16 +15,33 @@ import EntityPageHorizontalTable from '@ncigdc/components/EntityPageHorizontalTa
 import BAMSlicingButton from '@ncigdc/components/BAMSlicingButton';
 import DownloadButton from '@ncigdc/components/DownloadButton';
 import AnnotationsLink from '@ncigdc/components/Links/AnnotationsLink';
+import ProjectLink from '@ncigdc/components/Links/ProjectLink';
+import CaseLink from '@ncigdc/components/Links/CaseLink';
+import FileLink from '@ncigdc/components/Links/FileLink';
 import Link from '@ncigdc/components/Links/Link';
 import Hidden from '@ncigdc/components/Hidden';
 import { userCanDownloadFile } from '@ncigdc/utils/auth';
 import { toggleFilesInCart } from '@ncigdc/dux/cart';
 import { setModal } from '@ncigdc/dux/modal';
 import Button from '@ncigdc/uikit/Button';
-import NoAccessModal from './Modals/NoAccessModal';
+import NoAccessModal from '@ncigdc/components/Modals/NoAccessModal';
 
 declare var GDC_AUTH: string;
 
+// value of data_category mapped to sections to display
+const DISPLAY_MAPPING = {
+  'Raw Sequencing Data': ['analysis', 'readGroup', 'downstreamAnalysis'],
+  'Transcriptome Profiling': ['analysis', 'downstreamAnalysis'],
+  'Simple Nucleotide Variation': ['analysis', 'downstreamAnalysis'],
+  'Copy Number Variation': ['analysis', 'downstreamAnalysis'],
+  'Structural Rearrangement': ['analysis', 'downstreamAnalysis'],
+  'DNA Methylation': ['analysis', 'downstreamAnalysis'],
+};
```

Comment:
it's for showing sections based on `data_category` to mimic the angular version. re: comment above.